OC4IDS_Data_Feedback_Notebook.ipynb

```diff
@@ -20,7 +20,7 @@
       "version": "3.8.2"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %%sql a &lt;&lt;
</pre><p>Avoid single character variable names, e.g. use <code>results</code> instead of <code>a</code>.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>